src/components/forms/forms.js

```diff
@@ -62,15 +69,30 @@ const normalizeValue = (value) => {
   return value;
 };
 
+const clearDependentFormFields = (config) => {
+  const {onChange, dependentFormFields = []} = config;
+  /*
+   * Clears fields for questions that are rendered conditionally, setting them to their default value
```

Comment:
I think of "clear" as to set it to blank, like `''`, while "reset" would return to the default value - There's a real value to having both of these functions! What would you think of adding a param to allow you to completely clear to `type.defaultValue`?

Reply:
Actually... I don't know, the more I'm thinking about it, the more I'm wondering if this is necessary! Should handling behavior of dependent form components be a job of this global forms? It feels like it's possible the parent formField handles clearing child form field values through the `onChange` option that's already available.